src/utils/format-endpoint-url.ts

```diff
@@ -0,0 +1,8 @@
+import type { Datex } from "datex-core-legacy/mod.ts";
+
+export function formatEndpointURL(endpoint:Datex.Endpoint) {
+	const endpointName = endpoint.toString();
+	if (endpointName.startsWith("@+")) return `${endpointName.replace("@+","")}.unyt.app`
+	else if (endpointName.startsWith("@@")) return `${endpointName.replace("@@","")}.unyt.app`
+	else if (endpointName.startsWith("@")) return `${endpointName.replace("@","")}.unyt.me`
```

Comment:
Is there a reason why not using `Datex.Endpoint.name` instead?

Reply:
no